### DISCUSSION TITLE
Chrome injection didn't work

### DISCUSSION THREAD

**Discussion by Kristey** [eliemichel/MapsModelsImporter#186] [Q&A]
When I click Inject to Process everything seems fine, but when I then click OK on the Google Chrome GPU box, RenderDoc simply returns to its home screen. 
I've also noticed that my Chrome doesn't show any debugging metrics. 
Any idea what I'm doing wrong?

I user this command:
C:\Windows\System32\cmd.exe /c "SET RENDERDOC_HOOK_EGL=0 && START "" ^"C:\Program Files\Google\Chrome\Application\chrome.exe^" --disable-gpu-sandbox --gpu-startup-dialog"

**Comment by eliemichel** [comment]
Your command seems right, so I don't know, does it do the same if you try with MS Edge instead of Chrome? Both work usually.

**Reply by SynergyWiz** [reply]
I seem to be having the same issue. The problem seems to be the browser loads instantly despite the command being correct. I tried Edge as suggested and it started to work. I got as far as the API in RenderDoc showing something beside "None", but then Edge did a refresh and it closed me out again. When tried to run the command again, Edge loaded instantly and didn't show the white box as before. 

Any suggestions would be awesome. I am planning on using this to 3D print city maps and sell them, and will be happy to donate to this project if I can get it to work.

Thanks for working on this project regardless!
